packages/example-getting-started/docs/2-scaffold-app.md

```diff
@@ -0,0 +1,13 @@
+### Create your app scaffolding
+
+Install the `@loopback/cli` package. This will give you the command-line
+toolkit that can generate a basic REST app for you.
+`npm i -g @loopback/cli`
+
+Next, navigate to whichever directory you'd like to create your new project
```

Comment:
I'm already confused here. The previous page tells me to download `example-getting-started` but we don't do anything with it? Maybe we should clarify somewhere that `example-getting-started` has only been downloaded for users' reference